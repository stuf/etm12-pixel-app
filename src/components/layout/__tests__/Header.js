import * as React from 'karet';
import { render } from 'enzyme';
import toJson from 'enzyme-to-json';

import LayoutHeader from 'components/layout/Header';

it('matches snapshot', () => {
  const w = render(
    <LayoutHeader
      {...{ menuItems: [], isEditing: false, env: {}, name: 'From test' }}
    />,
  );

  expect(toJson(w)).toMatchSnapshot('LayoutHeader minimal data');
});
