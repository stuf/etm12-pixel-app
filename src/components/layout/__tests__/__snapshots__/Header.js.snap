// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot: LayoutHeader minimal data 1`] = `
<header
  class="layoutHeader"
>
  <div
    class="layoutHeader__logoWrapper"
  >
    <img
      alt="Logo"
      class="layoutHeader__logo"
      src="logo.svg"
    />
  </div>
  <div
    class="layoutHeader__wrapper"
  >
    <div
      class="flex-horizontal -vertical-center -justify-end pad -p-right"
    />
  </div>
</header>
`;
